doc/source/learning/openstack-helm.rst

```diff
@@ -0,0 +1,116 @@
+Openstack Helm
+=====================
+
+Openstack-helm is a collection of packages to deploy Opentack in kubernetes.
+
+These packages are written using the helm packaging structure called Chart that describe how to
+automatize the deployment the images.
+
+The Openstack images are built using LOCI_, rest of images
+are created using docker. See `role SUSE-build-images`_
```

Comment:
(nit) This sounds like LOCI is doing something other than docker which could be a little confusing